src/test/java/com/solvd/meta/carina/LoginTest.java

```diff
@@ -25,4 +21,14 @@ public void loginTest() {
         InventoryPage inventoryPage = loginPage.clickLogIn();
         Assert.assertTrue(inventoryPage.isPageOpened(), "Login failed after click login button");
     }
+
+    @Test
+    public void loginLockedUserTest() {
+        LogInPage loginPage = new LogInPage(getDriver());
+        loginPage.open();
+        Assert.assertTrue(loginPage.isPageOpened(), "Home page is not opened");
+        loginPage.setUsername(R.TESTDATA.get("locked_user"));
+        loginPage.setPassword(R.TESTDATA.get("standard_user_password"));
+        Assert.assertTrue(loginPage.isPageOpened(), "Login aproved after click login button");
```

Comment:
test doesn't verify that user is not able to login (click on login button is missing), add it please
Also let's add one more verification. Verify that this banner appears after click 
<img width="352" alt="image" src="https://user-images.githubusercontent.com/103670717/165318190-b62b0f64-a876-4d53-9b92-501ef7fb9e33.png">